### PR TITLE
Check for the correct Java post installation

### LINKED ISSUE #123: Debian packages lack dependency on java
Debian packages are missing a required dependency on java. Following installation steps on a blank debian minimal works but daemon is unable to start because java is missing.

I think a "Depends: java8-runtime-headless" should be added.

### PR DESCRIPTION
**Pull request by BClark09**
closes #123 

Signed-off-by: Ben Clark <ben@benjyc.uk>

### CODE REVIEW

File: resources/control-runtime/postinst [hunk 1]
```diff
@@ -87,6 +87,16 @@ restartIfFlagFileExists(){
   return 0
 }
 
+checkJava(){
+  # Java must be at least version 8, check for this
+  . /usr/share/openhab2/runtime/bin/inc
+  locateJava >/dev/null
+  VERSION=$("${JAVA}" -version 2>&1 | awk -F '"' '/version/ {print $2}' | sed -e 's/_.*//g; s/^1\.//g; s/\..*//g; s/-.*//g;')
+  if [ "${VERSION}" -lt "8" ]; then
+    echo "### WARNING: Java 8 not detected! You must install Java 8 in order to use openHAB 2.X.X"
```
Comment (ThomDietrich) [inline resources/control-runtime/postinst]: Heyho! How about we make those printouts a bit friendlier?

> ### WARNING: We were not able to detect a Java 8 runtime on your system. It is a prerequisite for openHAB 2 and is needed before openHAB can be executed. Please install a current Java 8 runtime now or check the openHAB documentation for details.

Comment (BClark09) [reply resources/control-runtime/postinst]: Hey @ThomDietrich !

Good idea, I changed the message but didn't think the messages from openHAB stood out much. What do you think of:

image

Comment (ThomDietrich) [reply resources/control-runtime/postinst]: I like it very much! I'd remove the `#` though. It could be confused with a root prompt...

Comment (BClark09) [reply resources/control-runtime/postinst]: Fair enough :) Thanks!

**Pull request merged**